neqo-transport/src/tracking.rs

```diff
@@ -351,6 +353,58 @@ pub struct AckToken {
     ranges: Vec<PacketRange>,
 }
 
+/// The counts for different ECN marks.
+#[derive(PartialEq, Eq, Debug, Clone, Default, Copy)]
+pub struct EcnCount(EnumMap<IpTosEcn, u64>);
```

Comment:
Do you really need a `Copy` implementation here?  This is 256 bits, which I believe is a bit larger than the usual cutoff for `Copy`.